static/sass/_snapcraft_snap-details.scss

```diff
@@ -1,9 +1,91 @@
 @mixin snapcraft-snap-details {
+  $asciinema-background: #121314;
+
   @media screen and (max-width: $breakpoint-small) {
     .snapcraft-snap-details {
       &__snippet {
         margin-left: -4.5rem !important;
       }
     }
   }
+
+  .p-snap-details__media {
+    // videos holder
+    .p-snap-details__media-items {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+
+      .p-carousel__item--screenshot {
+        display: flex;
+        flex-grow: 0;
+        height: auto;
+        margin-bottom: $spv-inter--regular;
+        max-height: 78px;
+        overflow: hidden;
+      }
+
+      .p-carousel__item-screenshot img,
+      .p-carousel__item-screenshot video {
+        align-self: center;
+        width: 100%;
+      }
+
+      @media screen and (max-width: $breakpoint-medium) {
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        margin-bottom: -$spv-inter--regular;
+        margin-right: -$sph-inter;
+        margin-top: $spv-inter--regular;
+
+        .p-carousel__item--screenshot {
+          margin-bottom: $spv-inter--regular;
+          margin-right: $sph-inter;
+          max-height: 100%;
+          max-width: 129px;
+        }
+      }
+
+      @media screen and (max-width: $breakpoint-x-small) {
+        .p-carousel__item--screenshot {
+          max-width: calc(50% - 1rem);
+        }
+      }
+    }
+
+    .p-snap-details__media-items--distributed {
+      @media screen and (min-width: $breakpoint-medium) {
+        justify-content: space-between;
+
+        .p-carousel__item--screenshot {
+          margin-bottom: 0;
+        }
+      }
+    }
+
+    iframe {
+      margin-bottom: 0;
+    }
+
+    .asciicast {
```

Comment:
What element has this class? Is it ours or rendered by asciinema? I can't find in in a DOM (even when I have asciinema embedded).

Reply:
Yeah it's the embedded asciinema holder that gets added just after the script tag (has the id `asciicast-container-{id}`

Reply:
Ah, found it now. I had a typo in selector. 'Asciinema' etc are so "human friendly" names... 😄